generators/client/templates/react/src/main/webapp/app/modules/administration/logs/logs.tsx.ejs

```diff
@@ -69,7 +69,7 @@ export class LogsPage extends React.Component<ILogsPageProps, ILogsPageState> {
     const loggers = logs ? logs.loggers : {};
     return (
         <div>
-          <h2><Translate contentKey="logs.title">Logs</Translate></h2>
+          <h2 id="page-heading"><Translate contentKey="logs.title">Logs</Translate></h2>
```

Comment:
since we are using a generic id on all pages may better to use it as className rather than id